test/tap/groups/ai-g1/env.sh

```
@@ -0,0 +1,16 @@
+# AI-g1 Subgroup Environment Configuration
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)